.eslintrc.json

```diff
@@ -13,6 +13,7 @@
   },
   "parser": "babel-eslint",
   "rules": {
-    "no-var": ["error"]
+    "no-var": ["error"],
+    "curly": ["error", "all"]
```

Comment:
Why?  Revert?  @ctavan Any opinion here?  Since we're using `prettier`, maybe avoid custom rules that are just about esthetics.

Reply:
re `prettier`: Enforcing curly braces [changes the AST and is therefore not in the scope of `prettier`](https://github.com/prettier/prettier/issues/4262#issuecomment-379000483).

Re this setting: I was actually hoping that the `"standard"` eslint config would give us a reasonable set of default rules like this one but apparently that's not the case. I personally like https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb but should we decide to include it we should do that in a different PR, that will go too far. It was probably a bad move from my end to propose adding this change to this PR in the first place, apologies! 🙇‍♂️